pkg/conf/schedulerconf.go

```diff
@@ -51,6 +51,9 @@ var once sync.Once
 var configuration *SchedulerConf
 var factory = initConfigs
 
+var Version string
+var Date string
+
```

Comment:
Add a boolean flag to show that it is the plugin version or not. Set to true in the schedulerplugin/main.go file